licorne/layer.py

```diff
@@ -37,33 +72,65 @@ def __init__(self,thickness=0.,
         To input the value, minimum and maximum, you should enter a 
         triplet (list,set, numpy array, etc)
         """
-        self.thickness=self._to_NumericPar(thickness)
-        self.nsld=self._to_NumericPar(nsld)
-        self.nsldi=self._to_NumericPar(nsldi)    
-        self.msld=MSLD(self._to_NumericPar(msld_rho),
-                       self._to_NumericPar(msld_theta),
-                       self._to_NumericPar(msld_phi))
-        self.roughness=self._to_NumericPar(roughness)
-        self.rougness_model=rougness_model
-        if self.rougness_model not in ['erf', 'tanh', 'NC']:
-            self.rougness_model=None
+        self.thickness=thickness
+        self.nsldr=nsldr
+        self.nsldi=nsldi
+        self.msld=(msld_rho,msld_theta, msld_phi)
+        self.roughness=roughness
+        self.roughness_model=roughness_model
+        if self.roughness_model not in ['erf', 'tanh', 'NC']:
+            self.roughness_model=None
```

Comment:
Same as below. Strings may be shown in the GUI but should **not** be used internally for the different models.